webapp/src/main/webapp/WEB-INF/gatein-resources.xml

```diff
@@ -103,6 +103,9 @@
       <depends>
         <module>webConferencingPortlet</module>
       </depends>
+      <depends>
+        <module>ManageVisioConnectorExtension</module>
```

Comment:
This should not be like that, you should add 
.finally(() => Vue.prototype.$utils.includeExtensions('xxx'));
in the js loading the webconferencing administration.
I add a suggestion.

Reply:
As I suggest the name ExternalVisioAdditionalActionsExtension for the js module, 
you should add 
.finally(() => Vue.prototype.$utils.includeExtensions('AdditionalActionsExtension'));

This line will load all js module named XXXAdditionalActionsExtension
So that, if we have more than one extension on this point, it will be also loaded.

Take example here : 
https://github.com/exoplatform/agenda/blob/develop/agenda-webapps/src/main/webapp/vue-app/agenda/main.js#L35